src/components/Landing/Landing.test.js

```diff
@@ -61,14 +61,14 @@ describe('Landing Component', () => {
     ).toBeTruthy();
   });
 
-  test('Contains a link to /courses and /howtTo', () => {
+  test('Contains a link to /courses and /howTo', () => {
     const { container } = render(
       <LastLessonProvider>
         <Landing {...PROPS} />
       </LastLessonProvider>
     );
 
-    const links = [...container.querySelectorAll('link')];
+    const links = [...container.querySelectorAll('a')];
 
     expect(validateElementAttributeValue(links, 'to', '/courses')).toBe(true);
     expect(validateElementAttributeValue(links, 'to', '/howTo')).toBe(true);
```

Comment:
Not part of this PR (and not blocking), but `validateElementAttributeValue` isn't a very clear name for what this function is doing. I was confused by this test, thinking "How could these elements all have a `to` prop that matches both `/courses` and `/howTo`? It wasn't until I looked into the function that it's looking for ANY element in a list to match the given prop. I'd suggest renaming this function to something like `someElementHasPropValue`

Reply:
Done!